src/components/ReportActionItem/TripRoomPreview.tsx

```diff
@@ -62,7 +63,7 @@ function ReservationView({reservation}: ReservationViewProps) {
     const {translate} = useLocalize();
 
     const reservationIcon = getTripReservationIcon(reservation.type);
-    const title = reservation.type === CONST.RESERVATION_TYPE.CAR ? reservation.carInfo?.name : reservation.start.longName;
+    const title = reservation.type === CONST.RESERVATION_TYPE.CAR ? reservation.carInfo?.name : Str.recapitalize(reservation.start.longName ?? '');
```

Comment:
maybe if we use this function always with the operator ?? we could make a new one that returns '' when it gets invalid arguments

Reply:
I didn't want to implement yet another minimal wrapper for this util. `longName` shouldn't be ever `undefined` for hotel reservation (and even if it was we are defaulting to '' Which ultimately is a falsy value - meaning we are safe).

Reply:
Thanks for this idea @sumo-slonik, but for the sake of minimalism I would leave it as is.